equinix/data_source_metal_device_acc_test.go

```diff
@@ -49,7 +49,7 @@ resource "equinix_metal_device" "test" {
   hostname         = "tfacc-test-device"
   plan             = local.plan
   metro            = local.metro
-  operating_system = "ubuntu_16_04"
+  operating_system = "ubuntu_22_04"
```

Comment:
Updated the `ubuntu_16_04` OS image references throughout the project to `ubuntu_22_04` because newer plans matched by `equinix_metal_plans` did not support 16.04 (device provisions were failing). 

I considered adding a datasource for `equinix_metal_operating_system` and making the `local.os` conditional on a `provisionable_on` and `distro = "ubuntu"` match, but found this quick fix to be much simpler.

Reply:
```
        Error: API Error HTTP 422 ubuntu_22_04 is not a valid operating system
```

Welp. That didn't work.